### PR TITLE
Replace SA builder with ci-operator

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: pkg/steps/release/create_release.go [hunk 1]
```diff
@@ -102,6 +105,61 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
 		return nil
 	}
 
+	if _, err := s.saGetter.ServiceAccounts(s.jobSpec.Namespace).Create(&coreapi.ServiceAccount{
+		ObjectMeta: meta.ObjectMeta{
+			Name:      "ci-operator",
+			Namespace: s.jobSpec.Namespace,
+		},
+	}); err != nil && !errors.IsAlreadyExists(err) {
+		return fmt.Errorf("could not create service account '%s' for: %v", "ci-operator", err)
```
Comment (stevekuznetsov) [inline pkg/steps/release/create_release.go]: Why bother print-formatting when it's a static string?

File: pkg/steps/release/create_release.go [hunk 2]
```diff
@@ -102,6 +105,61 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
...
+				Resources: []string{"imagestreams/layers"},
+				Verbs:     []string{"get", "update"},
+			},
+			{
+				APIGroups: []string{"", "image.openshift.io"},
+				Resources: []string{"imagestreams", "imagestreamtags"},
+				Verbs:     []string{"create", "get", "list", "update", "watch"},
+			},
+			{
+				APIGroups: []string{"", "build.openshift.io"},
```
Comment (stevekuznetsov) [inline pkg/steps/release/create_release.go]: We will not need this

File: pkg/steps/release/create_release.go [hunk 3]
```diff
@@ -102,6 +105,61 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
...
+				Resources: []string{"builds/details"},
+				Verbs:     []string{"update"},
+			},
+			{
+				APIGroups: []string{"", "build.openshift.io"},
```
Comment (stevekuznetsov) [inline pkg/steps/release/create_release.go]: Or this

File: pkg/steps/release/create_release.go (hunk 1 above)
Comment (hongkailiu) [reply pkg/steps/release/create_release.go]: Done.

File: pkg/steps/release/create_release.go (hunk 2 above)
Comment (hongkailiu) [reply pkg/steps/release/create_release.go]: Removed.

File: pkg/steps/release/create_release.go (hunk 3 above)
Comment (hongkailiu) [reply pkg/steps/release/create_release.go]: Removed

**Comment by hongkailiu** [comment]
> Shouldn't this impact integration tests?

I am also surprised the tests passed. Checking.

**Comment by stevekuznetsov** [comment]
Don't change the TODO.

**Comment by petr-muller** [comment]
Not being day-to-day in context in which this change is needed, I am really missing a **why** in the description of this PR.

**Comment by hongkailiu** [comment]
Background: https://coreos.slack.com/archives/CHY2E1BL4/p1574735871289900?thread_ts=1574733720.287000&cid=CHY2E1BL4

**Comment by droslean** [comment]
/lgtm
/hold
@petr-muller Wanna check?

**Comment by hongkailiu** [comment]
Witnessed by 
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/integr8ly_integreatly-operator/243/pull-ci-integr8ly-integreatly-operator-master-e2e/422

```bash
$ oc get rolebinding -n ci-op-k10ph71r ci-operator-image
NAME                ROLE                               USERS   GROUPS   SERVICE ACCOUNTS   SUBJECTS
ci-operator-image   ci-op-k10ph71r/ci-operator-image                    ci-operator  
$ oc get role -n ci-op-k10ph71r ci-operator-image
NAME
ci-operator-image
$ oc get sa -n ci-op-k10ph71r ci-operator 
NAME          SECRETS   AGE
ci-operator   2         20m

```

**Pull request merged**